assets/build/store.js

```
@@ -1 +1,1245 @@
+const reducer = (state = DEFAULT_STATE, action) => {
+  switch (action.type) {
+    case 'SET_DOCS':
+      return {
+        ...state,
+        docs: [...action.docs]
+      };
+    case 'SET_DOC':
+      const setDocState = {
+        ...state,
+        docs: [...state.docs, action.doc]
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)